src/frontend/shortlist/src/App.vue

```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function markSchoolAsAccepted(payload) {
+  console.log(payload);
+  let apiReq = {
+    recoID: payload.recoID,
+    accepted: payload.accepted,
+  };
+  let success = () => {
+    console.log("Added to a list successfully");
+  };
+  let failure = (err) => {
+    console.log("Fail to add to a list.", err.response.data);
+  };
+  let req = apiClient.removeSchoolCard(apiReq, success, failure);
```

Comment:
Are you using the same function (removeSchoolCard) for both markAsAccepted and markAsRejected? If so I would change the name of it to something like updateRecommendation. And in the actual function you can use data:payload instead so that it will take whatever you pass in, trashed or accepted.

Reply:
Updated